stdlib/2and3/ctypes/wintypes.pyi

```diff
@@ -150,7 +155,8 @@ class WIN32_FIND_DATAW(Structure):
     cFileName: Array[WCHAR]
     cAlternateFileName: Array[WCHAR]
 
-# These pointer type definitions use _Pointer instead of POINTER, to allow them to be used in type annotations.
+# These pointer type definitions use _Pointer instead of POINTER, to allow them to be used
```

Comment:
So if a user wants to annotate something as pointer will it look like
```python
from ctypes import _Pointer, c_int
x: _Pointer[c_int]
```
Or is it a purely internal type that should never be used? If not maybe we can pretend `pointer` is a class (even if it is a function at runtime)?

Reply:
The real `ctypes._Pointer` is the internal base class for all pointer types, and is not supposed to be used directly. But you're right, if the `_Pointer` from the stub is private, there's no way for users to annotate something as a pointer (and that is definitely a use case that should be supported). I like the idea of making `pointer` a class in the stub, I'll try that and see if it works out.